api/src/main/java/org/apache/iceberg/RewriteManifests.java

```diff
@@ -39,6 +39,8 @@
    * with the same cluster key will be written to the same manifest (unless the file is large and
    * split into multiple files).
    *
+   * Note that either clusterBy/rewriteIf should be used or deleteManifest/addManifest, not both.
+   *
```

Comment:
New lines in Javadoc don't create new paragraphs, so you need to add `<p>` on each one.